data/colors/vars/deprecated_light.ts

```diff
@@ -1,583 +0,0 @@
-import {alpha, get} from '../../../src/utils'
-
-// Variables to be removed in the next major release
-
-export default {
```

Comment:
What if we had this file export an empty object instead of deleting it entirely? That way we have a place to move deprecated variables in the future.